zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -377,28 +376,6 @@ public void parseProperties(Properties zkProp) throws IOException, ConfigExcepti
                 backupConfigBuilder.setTimetableStoragePath(value);
             } else if (key.equals(BackupSystemProperty.BACKUP_TIMETABLE_BACKUP_INTERVAL_MS)) {
                 backupConfigBuilder.setTimetableBackupIntervalInMs(Long.parseLong(value));
-            } else if (key.equals(X509AuthenticationConfig.SET_X509_CLIENT_ID_AS_ACL)) {
```

Comment:
any reason why you are deleting all the following lines?

Reply:
The description of this PR has more details on this.